src/lightning_lite/lite.py

```diff
@@ -159,19 +166,77 @@ def setup(
             model = self._move_model_to_device(model=model, optimizers=list(optimizers))
 
         # Let accelerator/plugin wrap and connect the models and optimizers
-        model, optimizers = self._strategy.setup_module_and_optimizers(model, list(optimizers))
+        if optimizers:
```

Comment:
If `setup` becomes `setup_module_and_optimizers`, then this `if` can be removed and we can reslve the typing mess